### DISCUSSION TITLE
Mobile support

### DISCUSSION THREAD

**Discussion by LucasD3** [vslinko/obsidian-zoom#16] [Ideas]
It would be great to have this functionality in Obsidian Mobile. While the ideal would be to have a touch interface, it would even be helpful just to be able to access zoom functionality via the command palette in mobile.

**Comment by FelipeRearden** [comment]
@vslinko forgive me for one more question :)

Are there plans for Obsidian support for Obsidian Mobile ?

I was looking to styles.css and I saw `is-mobile` and just wanna ask if we could get this amazing plugin on Mobile.

Thanks for reading this :)

**Reply by vslinko** [reply]
@FelipeRearden plug-in is ready for mobile and will be released when Obsidian Mobile will be updated to be compatible with Obsidian 0.13

**Reply by FelipeRearden** [reply]
Wooowww great news !!!!!

Thank very much for this @vslinko !!!!!

**Reply by FelipeRearden** [reply]
> @FelipeRearden plug-in is ready for mobile and will be released when Obsidian Mobile will be updated to be compatible with Obsidian 0.13


Hello again @vslinko !!!!

I try to use the new version `0.3.0` on Mobile (iPadOS) to see if I could use this amazing plugin on Mobile

I Changed the `manifest.json` :
⦿ `iSDesktopOnly` to `false`
⦿ `MinAppVersion` to `0.14.15` which is the lasted version to Mobile 

Unfortunatly it was not possible to enable the plugin on Mobile :(

There is a plugin [Mobile Logging](https://github.com/shabegom/obsidian-mobile-logging) that copy the **Console log** into a note on Obsidian Mobile. 

This way I could show to you the error that we get when trying to enable plugin on Mobile

![3FDAC0DA-F273-436D-8295-D24750872042](https://user-images.githubusercontent.com/85180696/181914495-7b2cc28b-f0f7-4fe4-bcae-d2a7114305fc.jpeg)

---

Would you mind taking a look to see if would be possible to add support for mobile ? 🙏🙏🙏

As someone who uses an iPad as a laptop (with mouse + keyboard) would be spectacular to be able to use this plugin on iPad as well 

Thanks for reading this and I wish you a fantastic day !!!!

**Comment by Fyzu** [comment]
Hi all, I added support for mobile devices, and would be happy to hear feedback :)

**Reply by fernandopirescoelho** [reply]
Wonderful!
This is really what I need!
How can we zoom on headings?

**Reply by Fyzu** [reply]
> Wonderful! This is really what I need! How can we zoom on headings?

Use command from toolbar panel
